stat/distmv/normal.go

```diff
@@ -277,6 +277,38 @@ func (n *Normal) Rand(x []float64) []float64 {
 	return x
 }
 
+// ScoreInput returns the score function with respect to the input of the
```

Comment:
I would rewrite these two sentences as: ScoreInput returns the value at x of the score function s which is defined as the gradient of the log-probability:  s(x) = \nabla log(p(x)).
If gradient is not used in statistics, then keep derivative.
Also, why not call the method just Score?